components/Navigation/index.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import Link from "next/link";
+import Icon from "@mdi/react";
+import { mdiPlusCircle } from "@mdi/js";
+import { mdiBullseyeArrow } from "@mdi/js";
+
+const NavBar = styled.nav`
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0px -1px 0px rgba(58, 72, 80, 0.07),
+    0px -2px 11px -1px rgba(176, 189, 197, 0.12);
+  padding: 15px 0 13px;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  gap: 75px;
+  justify-content: center;
+`;
+
+const StyledIconBox = styled(Link)`
+  transition: all 0.1s ease-in-out;
+  font-size: 11px;
+  line-height: 12px;
+  color: #0f0f0f;
+  cursor: pointer;
+  -webkit-tap-highlight-color: transparent;
```

Comment:
What does this style do?

Reply:
Hi @FelineHuhn 
I read that there might appear a highlight effect when clicking on a clickable element on a touchscreen. If you set -webkit-tap-highlight-color to transparent this highlight effect will not be visible. 
I never used it before, so I'm not sure if you can really see any difference😅

Reply:
Maybe I am wrong, but I didn't see an effect. You can test it out again in your next feature, and when there's no effect, it's better to remove this useless style.